modules/olm-bundle-format.adoc

```diff
@@ -59,11 +59,19 @@ etcd
 === Optional objects
```

Comment:
Is "supplemental objects" a better name for these? Just saw them referred to that way in a presentation.

Reply:
cc @exdx

Reply:
I would refer to them as "Additionally supported objects"